Examples/TicTacToe/Sources/Views-SwiftUI/LoginSwiftView.swift

```diff
@@ -15,6 +17,16 @@ public struct LoginView: View {
     var isLoginButtonDisabled: Bool
     var password: String
     var isTwoFactorActive: Bool
+
+    init(state: LoginState) {
```

Comment:
Should this be in an extension? Just thinking that in isowords it's convenient to be able to create ViewState independent of having domain State first, and might be nice for examples to encourage that as well.

Reply:
down to do that, but in practice we seem to always have the `init` up in the `struct` and I think that makes it easier to make changes in the future.

Reply:
I just seem to remember a recent refactor we did made GameOverView previews a lot easier to mock out by having the default init around. Or do you mean define the member wise one too?